test/e2e/feature/feature.go

```diff
@@ -137,6 +137,18 @@ var (
 	// Testing downward API huge pages
 	DownwardAPIHugePages = framework.WithFeature(framework.ValidFeatures.Add("DownwardAPIHugePages"))
 
+	// owning-sig: sig-scheduling
+	// kep: https://kep.k8s.io/4815
+	// test-infra jobs:
+	// - "ci-kind-dra-all" in https://testgrid.k8s.io/sig-node-dynamic-resource-allocation
+	//
+	// This label is used for tests which need:
+	// - the DynamicResourceAllocation *and* DRAPartitionableDevices feature gates
+	// - the resource.k8s.io API group
+	// - a container runtime where support for CDI (https://github.com/cncf-tags/container-device-interface)
+	//   is enabled such that passing CDI device IDs through CRI fields is supported
+	DRAPartitionableDevices = framework.WithFeature(framework.ValidFeatures.Add("DRAPartitionableDevices"))
```

Comment:
for just the featuregate you don't need this: https://groups.google.com/a/kubernetes.io/g/dev/c/uVga86nJDuI

it probably makes more sense to keep using the DRA e2e feature for DRA support in the runtime and then use WithFeatureGate for the specific gates

Reply:
At the moment, the DRA kind jobs select tests based on `Feature:`, not `Alpha/Beta`. It should be possible to change that, let me think about it.

Reply:
Related: I want to get to the point where some DRA tests don't need the `Feature:DynamicResourceAllocation`. This should be possible for those which don't need a registered DRA driver and actual support for DRA in the container runtime.

Reply:
https://github.com/kubernetes/kubernetes/pull/131691, https://github.com/kubernetes/test-infra/pull/34775